README.md

```diff
@@ -52,13 +55,21 @@ $ ./scripts/server
 
 ### Running directly
 
+* Make sure you have `nvm` and `yarn` installed (see links in Requirements)
 * `cd taui`
-* Install packages: `yarn add`
+* Specify Yarn and Node versions: Run `nvm use` and `yvm use`
```

Comment:
```suggestion
* Specify Yarn and Node versions: Run `nvm use` and `yvm use` (you may need to run `nvm install` or `yvm install` first if you see error messages that the appropriate versions are not installed).
```
I ran into an issue the first time trying to use this because I didn't have the correct Node version already installed.